Migrate from Microsoft.DBforPostgreSQL/servers to Microsoft.DBforPostgreSQL/flexibleServers

ganeshiyer-ms opened the pull request:
Uses `Microsoft.DBforPostgreSQL/flexibleServers` instead of  `Microsoft.DBforPostgreSQL/servers`. This is a more cost effective way of deploying postgresql on azure also is available in more regions.

ganeshiyer-ms committed:
migrated postgres to flexible/postgres

ms-RistoRK commented on src/arm/xnat.json:
```diff
@@ -249,7 +250,7 @@
                                 "requests": {
                                     "cpu": 4,
-                                    "memoryInGb": 16
+                                    "memoryInGB": 16
```
Is changing the capitalization correct? my concern is that the system is capitalization-sensitive and this may not be recognized and hence triggering an error.   If it not capitalization-sensitive then all good :)

ganeshiyer-ms replied:
yes this is how it's defined in the JSON schema: https://schema.management.azure.com/schemas/2018-04-01/Microsoft.ContainerInstance.json

ms-RistoRK commented on src/arm/xnat.json:
```diff
@@ -268,11 +269,11 @@
                     "ports": [
                         {
-                            "protocol": "tcp",
+                            "protocol": "TCP",
```
Is changing the capitalization correct? my concern is that the system is capitalization-sensitive and this may not be recognized and hence triggering an error. If it not capitalization-sensitive then all good :)

ganeshiyer-ms replied:
yes, this is defined in the JSON schema for Microsoft.ContainerInstance 
![image](https://user-images.githubusercontent.com/89279620/236270821-1f2e4dce-baad-4116-afa3-d981a09d6af8.png)

ref: https://schema.management.azure.com/schemas/2018-04-01/Microsoft.ContainerInstance.json

ms-RistoRK replied:
Ship it! :)

ms-RistoRK reviewed (dismissed):
I only have a few comments regarding capitalization.  If the capitalization does not impact then LGTM.

ganeshiyer-ms commented:
> I only have a few comments regarding capitalization. If the capitalization does not impact then LGTM.

Thanks for reviewing. I've addressed your comments.

ganeshiyer-ms committed:
updated .ui.json

ms-RistoRK reviewed (approved):
LGTM, ship it!
